global_schema.ddl

```diff
@@ -0,0 +1,294 @@
+/*
+ * Clouds
+ */
+CREATE TABLE IF NOT EXISTS cloud (
+	uuid				TEXT PRIMARY KEY,
+	name				TEXT,
+	type				TEXT,
+	endpoint			TEXT,
+	identity_endpoint	TEXT,
+	storage_endpoint	TEXT,
+	skip_tls_verify		BOOLEAN
+)
+
+CREATE TABLE IF NOT EXISTS cloud_region (
+	uuid				TEXT PRIMARY KEY,
+	cloud_uuid			TEXT NOT NULL,
+	name				TEXT,
+	endpoint			TEXT,
+	identity_endpoint	TEXT,
+	storage_endpoint	TEXT,
+	CONSTRAINT			cloud_region_cloud
+		FOREIGN KEY		(cloud_uuid)
+		REFERENCES		cloud(uuid)
+);
+
+CREATE TABLE IF NOT EXISTS auth_type (
+	uuid		TEXT PRIMARY KEY,
```

Comment:
We should change this. UUIDs are good keys for when you don't know the data that will be in the table. Here it will be a look-up defined by us so let's make this an integer ID.

We'll populate it right here with:
- oauth1
- userpass
- certificate
- access-key